### ISSUE TITLE
Cannot install using Go modules

### ISSUE THREAD

**Issue by carldunham** [go-chi/jwtauth#42]
At least with Go 1.13, I am not able to use 4.0.x versions. It's the same problem that chi v4 has, but there are changes in v4 here that I'm relying on. It looks like chi v4 was just an experiment in moduling.

**Comment by carldunham** [comment]
And by "relying on", I really mean "too lazy to work around", which I actually already have. Just leaving this here as evidence that more people are migrating to modules (as you apparently have for your own dependency management), and that this might be something to give higher priority to. It does seem like the "modules >= v2" problem has not been addressed well. Hopefully, you can find a clean way to deal with it. Happy to help if I can.

**Comment by VojtechVitek** [comment]
`GO111MODULE=on go get -u github.com/go-chi/jwtauth@v4.0.2` should work

**Comment by carldunham** [comment]
Sadly, that does not:
```
% GO111MODULE=on go get -u github.com/go-chi/jwtauth@v4.0.2
go: finding github.com/go-chi/jwtauth v4.0.2
go: finding github.com/go-chi/jwtauth v4.0.2
go get github.com/go-chi/jwtauth@v4.0.2: github.com/go-chi/jwtauth@v4.0.2: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v4
```

**Comment by VojtechVitek** [comment]
Interesting. I didn't realize I was using `v3.2.0+incompatible` in my codebase so far.

**Comment by VojtechVitek** [comment]
You're right. This project shouldn't have go.mod file, since it's not released with a proper "/v4" module path.

You'll have to use exact commit SHA when installing this pkg for now.. 

v4.0.2 has SHA 47840abb19b36dcf4d3ab4123b605b28ea511bc8, see https://github.com/go-chi/jwtauth/releases

and this works:
```
GO111MODULE=on go get -u github.com/go-chi/jwtauth@47840abb19b36dcf4d3ab4123b605b28ea511bc8
```

**Comment by pkieltyka** [comment]
hey guys, I've resolved this with jwtauth v4.0.3 -- that is, I've removed the invalid go.mod file.

also see https://github.com/go-chi/jwtauth/pull/38#issuecomment-533917879

**Comment by carldunham** [comment]
Hey, thanks for this. Working great now!
